frontend/src/metabase/components/ErrorPages/ErrorPages.styled.tsx

```diff
@@ -1,15 +1,13 @@
 import styled from "@emotion/styled";
 
-import { color } from "metabase/lib/colors";
-
 export const ErrorPageRoot = styled.div<{ bordered?: boolean }>`
   display: flex;
   width: 100%;
   height: 100%;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  ${({ bordered }) => bordered && `border: 1px solid ${color("border")};`}
+  ${({ bordered }) => bordered && `border: 1px solid var(--mb-color-border);`}
```

Comment:
In a situation like this, I'd put the property in the front so it's obvious what this line is about, then conditionally set the value based on prop instead. If we set the value to `false` the property will removed anyway.

Reply:
I know. That's why reviews exist. You can incorporate changes as you see fit.